lib/paperclip/media_type_spoof_detector.rb

```diff
@@ -76,6 +76,7 @@ def type_from_file_command
       Paperclip.run("file", "-b --mime :file", file: @file.path).
         split(/[:;\s]+/).first
     rescue Terrapin::CommandLineError
+      Paperclip.log("Problem getting type from `file` command.  Possible that `file` doesn't exist on this system.  Content Type validations don't work without this.  Possibly other stuff. YMMV.  Yay!")
```

Comment:
@josh-m-sharpe  
if you can remove `  Possibly other stuff. YMMV.  Yay!`, we can then merge it in. thanks!